aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -293,7 +293,7 @@ Tensor cumprod_backward(const Tensor& grad, const Tensor& input, int64_t dim) {
     AT_ASSERT(omitted_products.size(dim) == dim_size - k);
 
     grad_input.select(dim, k).copy_(
-        at::sum(grad.slice(dim, k) * omitted_products,dim));
+        at::sum(grad.slice(dim, k) * omitted_products.conj(), dim));
```

Comment:
I think you also need to modify this: https://github.com/pytorch/pytorch/blob/33aaea912abbe22fb6518807b5792d1c6bdc6829/torch/csrc/autograd/FunctionsManual.cpp#L422 to `return grad * (exclusive_normal * exclusive_reverse).conj();`?

Reply:
These changes are already in the PR.